sound/soc/sof/intel/pci-tgl.c

```diff
@@ -98,18 +98,18 @@ static const struct sof_dev_desc ehl_desc = {
 	.dspless_mode_supported	= true,		/* Only supported for HDaudio */
 	.default_fw_path = {
 		[SOF_IPC] = "intel/sof",
-		[SOF_INTEL_IPC4] = "intel/avs/ehl",
+		[SOF_INTEL_IPC4] = "intel/sof-ipc4/ehl",
```

Comment:
It does not really matter, EHL is in the TGL family and if ever it will get IPC4 support in SOF (it can be built, right, it is just the signing) then it must follow the convention we have.

Reply:
Or if we drop EHL entirely then sure, the driver can drop it, but do we want that?